lib/src/components/parallax_component.dart

```diff
@@ -34,30 +34,40 @@ extension ParallaxComponentExtension on Game {
 /// A full parallax, several layers of images drawn out on the screen and each
```

Comment:
why we need `loadParallaxComponent` extension on Game? can't you just do ParallaxComponent.load? why a function just to call another one, w/o any difference? not sure I understand that

Reply:
That is something that we have agreed sometime now, when we did the sprite/animation refactor, we have the global cache (Flame.images) and the cache on the game class, all static load functions receive an instance of Images, with a default to the global cache.

This extension enforces the use of the game cache, so all assets loaded by functions like that are guaranteed to have used the game cache.

Reply:
oh I see the images: images now